serde_v8/magic/v8slice.rs

```diff
@@ -71,7 +71,9 @@ pub(crate) fn to_ranged_buffer<'s>(
   value: v8::Local<v8::Value>,
 ) -> Result<(v8::Local<'s, v8::ArrayBuffer>, Range<usize>), v8::DataError> {
   if value.is_array_buffer_view() {
-    let view: v8::Local<v8::ArrayBufferView> = value.try_into()?;
+    // SAFETY: is_array_buffer_view() check above
+    let view: v8::Local<v8::ArrayBufferView> =
+      unsafe { v8::Local::cast(value) };
```

Comment:
Is this a performance optimization?

Reply:
Yes, it's avoiding any call to `is_array_buffer_view` https://docs.rs/v8/latest/src/v8/data.rs.html#903